core/trino-parser/src/test/java/io/trino/sql/TestSqlFormatter.java

```diff
@@ -291,7 +291,8 @@ public void testCreateTable()
                         FAIL,
                         ImmutableList.of(),
                         Optional.empty())))
-                .isEqualTo("""
+                .isEqualTo(
+                        """
                         CREATE TABLE test (
                            col VARCHAR NOT NULL WITH (abc = 'test', xyz = DEFAULT)
                         )""");
```

Comment:
Closing """ should go on a separate line

Reply:
Actually you cannot do it here. It will be a different string then.
Now it does not have trailing endline.
```
                        """
                        CREATE TABLE test (
                           col VARCHAR NOT NULL WITH (abc = 'test', xyz = DEFAULT)
                        )
                        """
```

would have trailing endline

Reply:
Or do you suggest
```
```
                        """
                        CREATE TABLE test (
                           col VARCHAR NOT NULL WITH (abc = 'test', xyz = DEFAULT)
                        )\
                        """
```

Reply:
Yes, the latter